### ISSUE TITLE
Text searching

### ISSUE THREAD

**Issue by chouguleds** [Ivshti/linvodb3#67]
@Ivshti  Is there any way to do the text search other than using $regex? Can I create a text index some how to do the fast text searching?

**Comment by Ivshti** [comment]
Yes, there is a way, but it's poorly documented at the moment: https://github.com/Ivshti/linvodb-fts

I don't have time to support it / document it.

another thing you can do is find some FTs search engine and plug linvodb data in there
